todolist/src/components/todoForm/TodoForm.jsx

```diff
@@ -20,9 +20,8 @@ const TodoForm = ({darkMode}) => {
     const handleSubmit = async () => {
         try {
             await axios.post('http://localhost:3000/todo/',
-                {
-                    title, description, startDate, endDate, status
-                }, {withCredentials: true});
+                {ToDoDto:{title: title, description: description, startDate: startDate, endDate: endDate, status: status}},
```

Comment:
Wird so immer noch nicht funktionieren, das backend mappt das automatisch auf das TodoDto. d.H. der Teil kann weg. Der Lesbarkeitshalber sollte das ganze dann noch in verschiedene Zeilen.